content/applications/hr/employees/new_employee.rst

```diff
@@ -198,94 +205,98 @@ Work Information tab
-  new location is added, and populates the field. For days the employee does not work, for example,
-  Saturday and Sunday, leave the field blank (:guilabel:`Unspecified`).
+  approving an :guilabel:`Expense`, a :guilabel:`Time Off` request, :guilabel:`Timesheet` entries,
+  and :guilabel:`Attendance` records for the employee. Hover over any of the selections to reveal
+  the :guilabel:`Internal Link` arrow. Click the :guilabel:`Internal Link` arrow to open a form with
+  the approver's :guilabel:`Name`, :guilabel:`Email Address`, :guilabel:`Company`,
+  :guilabel:`Phone`, :guilabel:`Mobile`, and :guilabel:`Default Warehouse` fields. These can be
+  modified, if needed. Use the breadcrumb menu to navigate back to the new employee form when done.
+
+  .. important::
+     The users that appear in the drop-down menu for the :guilabel:`Approvers` section must have
+     *Administrator* rights set for the corresponding human resources role. To check who has these
+     rights, go to :menuselection:`Settings app --> Users --> → Manage Users`. Click on an employee,
+     and check the :guilabel:`Human Resources` section of the :guilabel:`Access Rights` tab.
+
+     - In order for the user to appear as an approver for :guilabel:`Expenses`, they must have
+       either :guilabel:`Team Approver`, :guilabel:`All Approver`, or :guilabel:`Administrator` set
+       for the :guilabel:`Expenses` role.
+     - In order for the user to appear as an approver for :guilabel:`Time Off`, they must have
+       either :guilabel:`Officer` or :guilabel:`Administrator` set for the :guilabel:`Time Off`
+       role.
+     - In order for the user to appear as an approver for :guilabel:`Timesheets`, they must have
+       either :guilabel:`Manager`, :guilabel:`Officer`, or :guilabel:`Administrator` set for the
+       :guilabel:`Payroll` role.
+
+- :guilabel:`Remote Work`: using the drop-down menu, select the default location the employee works
+  from, for each day of the week. The default options are :guilabel:`Home`, :guilabel:`Office`, or
+  :guilabel:`Other`. A new location can be typed into the field, then click either :guilabel:`Create
+  (new location)` to add the location, or :guilabel:`Create and edit...` to add the new location and
+  edit the form. After edits are done, click :guilabel:`Save & Close`, and the new location is
+  added, and populates the field. For days the employee does not work, for example, Saturday and
+  Sunday, leave the field blank (:guilabel:`Unspecified`).
 
   .. note::
      It is also possible to add or modify work locations by navigating to :menuselection:`Employees
-     app --> Configuration --> Employee: Work Locations`. Click :guilabel:`New` to create a new
-     location, then enter the details on the form. To modify a location, click on an existing
-     location, then make any changes on the form.
+     app --> Configuration --> Employee: Work Locations`. To modify a location, click on an existing
```

Comment:
I can't comment on this line, but on 230 - 231, in the important admonition block
"To check who has these rights, go to..." There's an extra arrow in the menuselection navigation
![image](https://github.com/odoo/documentation/assets/23611711/8c45d593-5e8f-4fd3-9aad-1b7e389532af)

Reply:
The extra arrow is because what you click on is actually the arrow with the words Manage Users. It's odd, but if you go into the settings, you'll see it.